object.c

```diff
@@ -3684,22 +3706,31 @@ rb_Array(VALUE val)
 
 /*
  *  call-seq:
- *     Array(arg)    -> array
+ *    Array(object) -> object or new_array
+ *
+ *  Returns an array converted from +object+.
+ *
+ *  If +object+ is:
  *
- *  Returns +arg+ as an Array.
+ *  - An array, returns +object+.
+ *  - +nil+, returns an empty array.
```

Comment:
`nil` behavior should not be mentioned separately, since it isn't treated separately.  The behavior for `nil` is the same for non-arrays (`nil.to_a # => []`).

Reply:
Fixed.